src/praisonai-bot/praisonai_bot/gateway/server.py

```
@@ -853,10 +854,29 @@ def __init__(
+            store_kind = getattr(self.config.session_config, "store", "sqlite")
+            if store_kind == "file":
+                self._session_store = DefaultSessionStore(session_dir=persist_path)
+                logger.info(f"Session persistence enabled (file/JSON store), directory: {persist_path or '~/.praisonai/sessions/'}")
+            else:
+                try:
+                    from praisonaiagents.session.sqlite_transcript_store import (
+                        SqliteTranscriptStore,
+                    )
+                    self._session_store = SqliteTranscriptStore(session_dir=persist_path)
+                    logger.info(f"Session persistence enabled (SQLite store), directory: {persist_path or '~/.praisonai/sessions/'}")
+                except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)